src/coreclr/debug/createdump/createdumppal.cpp

```diff
@@ -0,0 +1,268 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "createdump.h"
+#include <time.h>
+#include <sys/time.h>
+
+#define INITGUID
+#include <guiddef.h>
+
+DEFINE_GUID(IID_IUnknown, 0x00000000, 0x0000, 0x0000, 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46);
```

Comment:
What needs this?

Reply:
There are a couple of QueryInterface functions that need it (CrashInfo impls ICLRDataEnumMemoryRegionsCallback and the data target impl).

Reply:
Yes.  The rest of the IIDs come from corguids library, but IID_IUnknown comes from the PAL.